client/stylesheets/_variables.scss

```diff
@@ -9,3 +9,21 @@ $gap-smallest: 4px;
 
 // @todo remove this spacing variable
 $spacing: 16px;
+
+// Gutenberg Button variables. So very temporary
+$default-font-size: 16px;
```

Comment:
Although these are temporary -- all but the `$blue-medium` variables are either defined in `_colors.scss` or via the font-size mixin.